sycl/test-e2e/Graph/Explicit/assume_buffer_outlives_graph_property.cpp

```diff
@@ -1,4 +1,5 @@
-// REQUIRES: level_zero, gpu
+// REQUIRES-ANY: opencl
+// REQUIRES-ALL: level_zero, gpu
```

Comment:
Reckon we could drop the gpu from this and shorten both lines to a single `REQUIRES: level_zero || opencl`

Reply:
I'm quite unfamiliar with `lit` and was unsure how to use conditionals here. I was trying `opencl || level_zero, gpu` but that was failing so found the above change worked but removing `gpu` makes things simpler and have changed all the tests to that. Thanks!